### ISSUE TITLE
Design for links that have google docs version

### ISSUE THREAD

**Issue by jstrothman** [18F/ux-guide#340]
### Point of contact on this issue
@bpdesigns or @MelissaBraxton 

______________________________________

**Timeline**
Does this need to happen in the next two weeks?

- [ ] Yes
- [X] No

**How much time do you anticipate this work taking?**
1-2 weeks

**Background**
The UX Guide has many links that are available as a web page as well as a Goog Doc version for internal reuse purposes. 

We use the following to distinguish these links:
`[link text for web-based link]. (18F/GSA access only 🔒 )`

There are a few problems with this: 
1. It doesn't pass [WCAG Success Criterion 2.4.4 Link Purpose (In Context)](https://www.w3.org/TR/WCAG22/#link-purpose-in-context): The purpose of each link can be determined from the link text alone or from the link text together with its programmatically determined link context, except where the purpose of the link would be ambiguous to users in general. Example problems this causes:
    * For screen reader users who pull up a list of links and listen to the links, they don't know what each `18F/GSA access only lock` goes to. This is a common screen reader user action.
    * It requires unnecessary effort for speech-to-text users who are trying to target the intended 'Read more' link
2. It's difficult to distinguish from links where we _only_ offer the GSA-only link. The only difference is the period and space at the end of the web-based link. I get daily emails with Drive requests for access to these docs. 

**Screenshot of similarity between 1 link and 2 links**
![Screen Shot 2023-01-10 at 1 18 14 PM](https://user-images.githubusercontent.com/1080991/211631570-b5e6ee1e-3416-40e3-85b1-feb1d734b595.png)

**Screenshot of screenreader links list with repeated ambiguous 18F/GSA access only links**
![ux-guide-links-list](https://user-images.githubusercontent.com/1080991/211632948-7bb223a7-2372-4cfb-b864-0dda0412002c.jpg)

**Acceptance criteria _(we'll know we're done when…)_** 
- [ ] The purpose of each link in the ux guide can be determined from the link text alone or from the link text together with its programmatically determined link context
- [ ] People easily know if they click on a link whether it is an internal and external links
- [ ] Considerations for applying the approach we take are documented in a place other guides/sites will find it

**Tasks**
- [ ] Design alternative for displaying links to alternate versions of the same content
- [ ] Design peer review
- [ ] Update all such links in the 18F Methods
- [ ] Update all such links in the UX Guide
- [ ] Tag UX Guide and Method product owners to review
- [ ] Document approach in a centralized place where other 18F products can find it

**Comment by bpdesigns** [comment]
@katefisher808 yes. The last item in the task list is documenting how this was done. One of the ways it could be documented is in another DA Team issue to review the handbook and other guides for similar situations.

**Comment by bpdesigns** [comment]
It makes me wonder if another outcome of "Document approach in a centralized place where other 18F products can find it" might be to create a wiki in the DA Team repo that shows all the work that relates to improving multiple repos. For example the 18F UX Guide documents [automations like syncing across repos](https://github.com/18F/ux-guide/wiki/Syncing-across-repos) that might be worth greater visibility in a DA Team repo

**Comment by katefisher808** [comment]
Thanks Ben! That makes sense. I'll follow-up on how to translate direction to other places once we narrow down to a direction.

I started a doc with initial exploration for the ux-guide as a starting point. I'm going to meet with a few 18F staff to get initial feedback on the suggestions to try to narrow down directions. 
https://docs.google.com/document/d/18a2YMYhthITego22yNfEhK9FpY9A4kmys1BbiuLfXk8/edit#

**Comment by katefisher808** [comment]
A heads up to watchers on this ticket. It is on hold until an engineer comes onto DA to support, which should be this week.

**Comment by katefisher808** [comment]
I'll also send this note on slack for visibility.

@bpdesigns Sarah W is on board to support from engineering team!

Update:
- From the design standpoint, We have narrowed to [three possible treatments for review](https://docs.google.com/document/d/18a2YMYhthITego22yNfEhK9FpY9A4kmys1BbiuLfXk8/edit).
- From the engineering standpoint, we are exploring how best to implement the future changes on github across the ux-guide.
- From a cross-product standpoint, we want to open new tickets for making changes to the handbook, other guides, and for documenting the change, so we can keep this ticket moving along

**Comment by bpdesigns** [comment]
Looks great! Treatment 3 stands out to me.

**Comment by katefisher808** [comment]
@bpdesigns phew! updates have been made to the ux-guide based on the recommendations and [quick discovery process](https://github.com/orgs/18F/projects/41/views/1?filterQuery=accessi&pane=issue&itemId=22192286#:~:text=three%20possible%20treatments%20for%20review). 
https://ux-guide.18f.gov/

Hats off to @geekygirlsarah on untangling a lot of the rules around the links.

Next steps: we are going to open new tickets for adjusting other guides, handbook and flag it for the respective product owners.
